test/Microsoft.Framework.Configuration.Test/ConfigurationTest.cs

```diff
@@ -12,20 +12,13 @@ namespace Microsoft.Framework.Configuration.Test
     public class ConfigurationTest
     {
         [Fact]
-        public void SetBasePathThroughConstructor()
+        public void SetBasePathThroughBuilder()
```

Comment:
Can you move this test to `Microsoft.Framework.Configuration.FileExtensions.Test`?